src/source/noise.rs

```diff
@@ -42,7 +42,7 @@ impl WhiteNoise {
     pub fn new(sample_rate: SampleRate) -> Self {
         Self {
             sample_rate,
-            rng: SmallRng::from_entropy(),
```

Comment:
what is the difference between these? Would `seed_from_u64(0)` not be better? then the noise is always the same.